src/MSBuild/XMake.cs

```diff
@@ -515,14 +516,7 @@ string commandLine
             ConsoleCancelEventHandler cancelHandler = Console_CancelKeyPress;
             try
             {
-#if (!STANDALONEBUILD)
-                // Enable CodeMarkers for MSBuild.exe
-                CodeMarkers.Instance.InitPerformanceDll(CodeMarkerApp.MSBUILDPERF,  String.Format(CultureInfo.InvariantCulture, @"Software\Microsoft\MSBuild\{0}", MSBuildConstants.CurrentProductVersion));
-#endif
-#if MSBUILDENABLEVSPROFILING 
-                string startMSBuildExe = String.Format(CultureInfo.CurrentCulture, "Running MSBuild.exe with command line {0}", commandLine);
-                DataCollection.CommentMarkProfile(8800, startMSBuildExe);
-#endif
+                XMakeEventSource.Log.XMakeStart("Execute MSBuild - Begin");
```

Comment:
Do the strings that are being passed into each event add any value?  Generally, you want the payload to any event to be specific to that event.  The payloads in the events that I've seen thus far are essentially duplicating the meaning of the event itself.